docs/query-language.md

```diff
@@ -341,6 +341,14 @@ have the attribute specified by `field` with one of the values specified in
 
 **Result Type:** array of key/count objects
 
+**Caveats:**
+* TopN on a frame with ranked cache will return the top bitmaps sorted by count.
+* Non-ranked frames maintain a LRU (least recently used) cache, thus a TopN query on this frame will return bitmaps sorted in most recently setbit order.
+* The frame's cache size determines the limit of sorted bitmaps.  
```

Comment:
The frame's cache size determines the number of sorted bitmaps to maintain in the cache for purposes of `TopN()` queries. There is a tradeoff between performance and accuracy; increasing the cache size will improve accuracy of results at the cost of performance.